diff --git a/dist/es/components/AnimatePresence/PopChild.mjs b/dist/es/components/AnimatePresence/PopChild.mjs
deleted file mode 100644
index f81cf3410b86473bf037d21d4296bd985daca808..0000000000000000000000000000000000000000
diff --git a/dist/es/components/AnimatePresence/PresenceChild.mjs b/dist/es/components/AnimatePresence/PresenceChild.mjs
index 5ce38b0e9f55ebcb289705ad77b337ea5b5314e6..1d8f50d75e78de3c3181082ca0fc847365bd19c6 100644
--- a/dist/es/components/AnimatePresence/PresenceChild.mjs
+++ b/dist/es/components/AnimatePresence/PresenceChild.mjs
@@ -1,10 +1,10 @@
 "use client";
 import { jsx } from 'react/jsx-runtime';
 import * as React from 'react';
-import { useId, useCallback, useMemo } from 'react';
+import { useId } from "@reach/auto-id";
+import { useCallback, useMemo } from 'react';
 import { PresenceContext } from '../../context/PresenceContext.mjs';
 import { useConstant } from '../../utils/use-constant.mjs';
-import { PopChild } from './PopChild.mjs';

 const PresenceChild = ({ children, initial, isPresent, onExitComplete, custom, presenceAffectsLayout, mode, }) => {
     const presenceChildren = useConstant(newChildrenMap);
@@ -49,9 +49,6 @@ const PresenceChild = ({ children, initial, isPresent, onExitComplete, custom, p
             onExitComplete &&
             onExitComplete();
     }, [isPresent]);
-    if (mode === "popLayout") {
-        children = jsx(PopChild, { isPresent: isPresent, children: children });
-    }
     return (jsx(PresenceContext.Provider, { value: context, children: children }));
 };
 function newChildrenMap() {
diff --git a/dist/es/components/AnimatePresence/use-presence.mjs b/dist/es/components/AnimatePresence/use-presence.mjs
index 9e1351fce82d5c1ef8d1744b6e9cf7a6e39f7924..62dc222d54299e8bcdc1c5300587574f4b74e5fe 100644
--- a/dist/es/components/AnimatePresence/use-presence.mjs
+++ b/dist/es/components/AnimatePresence/use-presence.mjs
@@ -1,4 +1,5 @@
-import { useContext, useId, useEffect, useCallback } from 'react';
+import { useId } from "@reach/auto-id";
+import { useContext, useEffect, useCallback } from 'react';
 import { PresenceContext } from '../../context/PresenceContext.mjs';

 /**
