diff --git a/dist/schemas.js b/dist/schemas.js
index 060d0b9..8a7c81f 100644
--- a/dist/schemas.js
+++ b/dist/schemas.js
@@ -26,6 +26,7 @@ const nextThemesSchema = z.strictObject({
   defaultTheme: z.string().default("system"),
   disableTransitionOnChange: z.boolean().default(true),
   forcedTheme: z.string().optional(),
+  scriptProps: z.custom().optional(),
   storageKey: z.string().default("theme")
 });
 const sidebarSchema = z.strictObject({
diff --git a/dist/style.css b/dist/style.css
index 387ae10..2a942b8 100644
--- a/dist/style.css
+++ b/dist/style.css
@@ -131,12 +131,14 @@
   }
 }
 @layer base {
-  *, ::after, ::before, ::backdrop, ::file-selector-button {
+  :where([class^="x:"], [class*=" x:"], [class^="nextra-"], [class*=" nextra-"]) {
+  &, &::after, &::before, &::backdrop, &::file-selector-button {
     box-sizing: border-box;
     margin: 0;
     padding: 0;
     border: 0 solid;
   }
+  }
   html, :host {
     line-height: 1.5;
     -webkit-text-size-adjust: 100%;
diff --git a/dist/types.generated.d.mts b/dist/types.generated.d.mts
index daa88cd..d8c22bc 100644
--- a/dist/types.generated.d.mts
+++ b/dist/types.generated.d.mts
@@ -109,6 +109,9 @@ interface LayoutProps {
          */
         disableTransitionOnChange?: boolean;
         forcedTheme?: string;
+        scriptProps?: React.ScriptHTMLAttributes<HTMLScriptElement> & {
+            [dataAttribute: `data-${string}`]: string;
+        };
         /**
          * @default "theme"
          */
